migration/validator-activation.md

```diff
@@ -0,0 +1,76 @@
+# Activation Guide
+
+This guide is part of the Nimiq PoW to PoS migration process and is intended for users who have already registered as validators during the Validator Registration Phase by **October 6th, 2024**. For registering a validator post-transition, refer to the [Becoming a Validator](/build/set-up-your-own-node/becoming-a-validator.md) guide. For more technical details on the migration, see [Migration Technical Details](migration-technical-details).
```

Comment:
It is also possible to use the activation tool as an "observer" (without registering previously) for those who want to still migrate the history or just one observers without taking participation in the voting for readiness